pyaedt/hfss.py

```diff
@@ -748,7 +748,23 @@ def create_circuit_port_between_objects(self, startobj, endobject, axisdir="XYNe
         str
             Name of port created when successful, ``False`` otherwise.
 
+        Examples
+        --------
+
+        Create two boxes that will be used to create a circuit port
+        named ``'CircuitExample'``.
+
+        >>> box1 = hfss.modeler.primitives.create_box([0, 0, 80], [10, 10, 5],
+        ...                                           "BoxCircuit1", "copper")
+        >>> box2 = hfss.modeler.primitives.create_box([0, 0, 100], [10, 10, 5],
+        ...                                           "BoxCircuit2", "copper")
+        >>> hfss.create_circuit_port_between_objects("BoxCircuit1", "BoxCircuit2",
+        ...                                                 hfss.AxisDir.XNeg, 50,
+        ...                                                 "CircuitExample", True, 50, False)
```

Comment:
```suggestion
        >>> hfss.create_circuit_port_between_objects("BoxCircuit1", "BoxCircuit2",
        ...                                          hfss.AxisDir.XNeg, 50,
        ...                                          "CircuitExample", True, 50, False)
```

White space suggestion.

Reply:
Done. Good catch!